aws/client.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import boto3
```

Comment:
Please add `boto3` to the [requirements](https://github.com/jina-ai/cloud-helper/blob/master/requirements.txt)